x/evm/types/access_list_tx_test.go

```diff
@@ -0,0 +1,162 @@
+package types
+
+import (
+	"math/big"
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/ethereum/go-ethereum/common"
+	ethtypes "github.com/ethereum/go-ethereum/core/types"
+	"github.com/stretchr/testify/suite"
+	"github.com/tharsis/ethermint/tests"
+)
+
+type AccessListTxTestSuite struct {
+	suite.Suite
+
+	sdkInt         sdk.Int
+	uint64         uint64
+	bigInt         *big.Int
+	overflowBigInt *big.Int
+	sdkZeroInt     sdk.Int
+	sdkMinusOneInt sdk.Int
+	invalidAddr    string
+	addr           common.Address
+	hexAddr        string
+}
+
+func (suite *AccessListTxTestSuite) SetupTest() {
+	suite.sdkInt = sdk.NewInt(100)
+	suite.uint64 = suite.sdkInt.Uint64()
+	suite.bigInt = big.NewInt(1)
+	suite.overflowBigInt = big.NewInt(0).Exp(big.NewInt(10), big.NewInt(256), nil)
+	suite.sdkZeroInt = sdk.ZeroInt()
+	suite.sdkMinusOneInt = sdk.NewInt(-1)
+	suite.invalidAddr = "123456"
+	suite.addr = tests.GenerateAddress()
+	suite.hexAddr = suite.addr.Hex()
+}
+
+func TestAccessListTxTestSuite(t *testing.T) {
+	suite.Run(t, new(AccessListTxTestSuite))
+}
+
+func (suite *AccessListTxTestSuite) TestAccessListTxCopy() {
+	tx := &AccessListTx{}
+	txCopy := tx.Copy()
+
+	suite.Require().Equal(&AccessListTx{}, txCopy)
+	// TODO: Test for different pointers
```

Comment:
address issue?

Reply:
TBH I don't know the necessity for this - this comment exists both in `TestLegacyTxCopy` as well as `TestDynamicFeeTxCopy`, so I figured this should apply here too.